data/supportserver/autoyast_supportserver_non_x86.xml

```diff
@@ -0,0 +1,90 @@
+<?xml version="1.0"?>
+<!DOCTYPE profile>
+<profile xmlns="http://www.suse.com/1.0/yast2ns" xmlns:config="http://www.suse.com/1.0/configns">
+  <deploy_image>
+    <image_installation config:type="boolean">false</image_installation>
+  </deploy_image>
+  <general>
+    <ask-list config:type="list"/>
+    <mode>
+      <confirm config:type="boolean">false</confirm>
+      <final_halt config:type="boolean">false</final_halt>
+      <final_reboot config:type="boolean">false</final_reboot>
+      <halt config:type="boolean">false</halt>
+      <second_stage config:type="boolean">true</second_stage>
+    </mode>
+    <proposals config:type="list"/>
+    <signature-handling>
+      <accept_file_without_checksum config:type="boolean">false</accept_file_without_checksum>
+      <accept_non_trusted_gpg_key config:type="boolean">false</accept_non_trusted_gpg_key>
+      <accept_unknown_gpg_key config:type="boolean">false</accept_unknown_gpg_key>
+      <accept_unsigned_file config:type="boolean">false</accept_unsigned_file>
+      <accept_verification_failed config:type="boolean">false</accept_verification_failed>
+      <import_gpg_key config:type="boolean">false</import_gpg_key>
+    </signature-handling>
+    <storage/>
+  </general>
+  <bootloader>
+    <general>
+      <timeout config:type="integer">-1</timeout>
```

Comment:
Are you sure -1 is correct value? Wouldn't it wait in grub forever? I would use 0 instead.

Reply:
-1 is used in other jobs which create images . take a look at disable_grub_timeout module . it works like this for past 5 month without issues. It will not wait forever , we two options :
1. openQA detect grub2 needle and select appropriate menu entry and proceed further
2. openQA fail to detect grub2 needle and fail in 80 second as it's defined in login module